docs/getting-started/azure-to-cluster/configure.rst

```diff
@@ -0,0 +1,85 @@
+.. _configure-azure-to-cluster:
+
+=================================
+Configure and deploy your cluster
+=================================
+
+After having subscribed to the offer, you can now start the process to deploy
+the cluster.
+
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _configure-azure-to-cluster_wizard:
+
+Configuration
+=============
+
+First it must be configured. On the *Home > Software as a Service*
+*(SaaS)* page, go to *Overview*. Now click on *Configure Account* at the top
+left of the Overview page.
+
+.. image:: ../../_assets/img/ConfigureAccount.png
+
+You'll now be redirected to the CrateDB Cloud Console.
+
+Once in the Console, authenticate with the Azure account you used to create the
+offer in the previous step. Select *Sign in with Azure AD* and continue.
+
+Now complete the 3 step wizard to deploy the cluster.
+
+
+Wizard step 1
```

Comment:
maybe a new subsection "Wizard" and then further subsections "Step one", "Step two", and "Step three"